spec/presenters/hyrax/collection_presenter_decorator_spec.rb

```diff
@@ -21,7 +21,7 @@
         def collection_type_badge
           "<span>"
         end
-        prepend Hyrax::CollectionPresenterDecorator
+        prepend Hyku::CollectionPresenterDecorator
```

Comment:
Just being a stickler but should the spec's path also be moved?

`spec/presenters/hyku/collection_presenter_decorator_spec.rb`

Reply:
I debated that but the spec is describing Hyrax::CollectionPresenter so I decided against it. 🤔

Reply:
I suppose an alternative approach is the change adventist's class name instead.

Reply:
I have no strong opinions!  The only reason why I was thinking that was because my switch to spec extension would probably not be able to find this because it sticks to convention so it would expect the same path pattern.